### DISCUSSION TITLE
Soft WDT reset happening on ESP8266

### DISCUSSION THREAD

**Discussion by tiagorosolen** [Networking-for-Arduino/EthernetENC#45] [Q&A] [answered]
Hi JAndrassy

Thank you for maintaining this lib. I'm a real fan and have used a number of times already. Unfortunately this time there's something I could not wrap my hand around. 

When I call **_client.write** -> **UIPClient::write** -> finally in the instruction **Enc28J60Network::writePacket** on line 205, it throws a WDT reset. It's random, sometimes it works.

I'm sending some big blocks (5000, 10000 chars) as html. The whole thing worked in arduino IDE but now on PlatformIO this is happening.

Just wanted to let you know and maybe (just maybe) you might have a light to shed over it.

Thanks again!

**Comment by JAndrassy** [comment, marked as answer]
I recommend the lwIP_enc28j60 bundled with the esp8266 platform
